src/flex-example-app-android.xml

```diff
@@ -79,6 +91,18 @@
 						</intent-filter>
 					</receiver>
 
+
+                    <receiver
+                        android:name="com.inmobi.commons.analytics.androidsdk.IMAdTrackerReceiver"
```

Comment:
odd... yeah it wouldn't start for me. if it works for you without it then lets keep it out.